libs/styles/iframe.css

```diff
@@ -28,6 +28,7 @@
 
 @media (max-width: 1200px) {
   .dialog-modal.commerce-frame .milo-iframe{
-    padding-bottom:100vh;
+    /* stylelint-disable-next-line unit-no-unknown */
```

Comment:
I'm not getting a lint error when removing this line.

Reply:
![image](https://github.com/adobecom/milo/assets/138484653/0b1bbf1c-b007-4bcb-a865-ed1b47b8d92c)
This is the error i'm getting when i remove the comment.

Reply:
I am getting the eslint error too.